src/java/org/apache/cassandra/index/sai/disk/vector/VectorPostingsWriter.java

```diff
@@ -24,12 +24,23 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.IntUnaryOperator;
 
 import org.apache.cassandra.io.util.SequentialWriter;
 import org.apache.cassandra.utils.Pair;
 
 public class VectorPostingsWriter<T>
 {
+
+    private final boolean useOrdinalsMapper;
```

Comment:
let's omit this in favor of making sure we always have a Mapper that does the right thing

Reply:
I need it for writeDeletedOrdinals.